sdk/core/azure-core/src/http/winhttp/win_http_transport.cpp

```diff
@@ -220,6 +220,28 @@ void WinHttpTransport::CreateSessionHandle(std::unique_ptr<_detail::HandleManage
     // ERROR_NOT_ENOUGH_MEMORY
     GetErrorAndThrow("Error while getting a session handle.");
   }
+
+// These options are only available starting from Windows 10 Version 2004, starting 06/09/2020.
```

Comment:
@danieljurek do you know if we have (or can get) any CI machines that test the newer OS so we can continuously build this code path?